content/en/docs/tasks/tools/_index.md

```diff
@@ -47,3 +47,13 @@ installed and configured.
 you need to do to get up and running with `kind`.
 
 <a class="btn btn-primary" href="https://kind.sigs.k8s.io/docs/user/quick-start/" role="button" aria-label="View kind Quick Start Guide">View kind Quick Start Guide</a>
+
+## kubeadm
+
+[`Kubeadm`](https://github.com/kubernetes/kubeadm) is a tool built to provide best-practice "fast paths" for creating Kubernetes clusters.
```

Comment:
We should link to the installing kubeadm guide at k8s.io instead.

Reply:
Do you mean change "https://github.com/kubernetes/kubeadm" to "/docs/setup/production-environment/tools/kubeadm/install-kubeadm/" ? 

If so, I still think here link to "https://github.com/kubernetes/kubeadm" is more suitable. 🤔